settings/Core.setting.php

```diff
@@ -515,6 +515,21 @@
     'description' => NULL,
     'help_text' => NULL,
   ),
+  'forceRecaptcha' => array(
+    'add' => '4.7',
+    'help_text' => NULL,
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'forceRecaptcha',
+    'type' => 'Boolean',
+    'quick_form_type' => 'YesNo',
+    'html_type' => '',
+    'default' => '0',
```

Comment:
I wonder if this is the right default for new installs - perhaps it should default to 1 (& maybe upgrade script to change over existing). Non blocking comment